pyEDAA/CLITool/GHDL.py

```diff
@@ -0,0 +1,88 @@
+#                                                                                                                      #
+# License:                                                                                                             #
+# ==================================================================================================================== #
+# Copyright 2017-2021 Patrick Lehmann - Boetzingen, Germany                                                            #
+# Copyright 2014-2016 Technische Universität Dresden - Germany                                                         #
+#                     Chair of VLSI-Design, Diagnostics and Architecture                                               #
+#                                                                                                                      #
+# Licensed under the Apache License, Version 2.0 (the "License");                                                      #
+# you may not use this file except in compliance with the License.                                                     #
+# You may obtain a copy of the License at                                                                              #
+#                                                                                                                      #
+#   http://www.apache.org/licenses/LICENSE-2.0                                                                         #
+#                                                                                                                      #
+# Unless required by applicable law or agreed to in writing, software                                                  #
+# distributed under the License is distributed on an "AS IS" BASIS,                                                    #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.                                             #
+# See the License for the specific language governing permissions and                                                  #
+# limitations under the License.                                                                                       #
+#                                                                                                                      #
+# SPDX-License-Identifier: Apache-2.0                                                                                  #
+# ==================================================================================================================== #
+#
+"""\
+This module contains the CLI abstraction layer for `GHDL <https://github.com/ghdl/ghdl>`__.
+"""
+from pyTooling.CLIAbstraction            import CLIOption
+from pyTooling.CLIAbstraction.Executable import Executable
+from pyTooling.CLIAbstraction.Argument   import CommandArgument, ShortFlagArgument, LongFlagArgument, LongValuedFlagArgument
```

Comment:
Maybe woth using multiple lines? `... import ( \n ... )`

Reply:
Done in 32688b10.